docker/compose/docker-compose.yml

```diff
@@ -63,8 +112,15 @@ services:
     networks:
       - default
   alertplugins:
+    image: mozdef/mozdef_alertplugins
+    build:
+      context: ../../
+      dockerfile: docker/compose/mozdef_alertplugins/Dockerfile
+      cache_from:
+        - mozdef/mozdef_alertplugins
+        - mozdef_alertplugins:latest
     restart: always
-    command: bash -c 'sleep 90 && source /opt/mozdef/envs/python/bin/activate && python alert_worker.py -c alert_worker.conf'
+    command: bash -c 'source /opt/mozdef/envs/python/bin/activate && python alert_worker.py -c alert_worker.conf'
```

Comment:
We unfortunately want to keep the sleep 90 command, basically it's a poor man's way to wait for elasticsearch service to get booted up and all ready.

Reply:
it seems to boot just fine, we also removed it in cloudymozdef
if not, i would use wait-for-it instead of sleep, which is docker's recommended way to do it as well. that said, in my tests i found no issue with not sleeping so far

Reply:
note: the py.test 10s wait is equally annoying during testing

Reply:
This can be improved by running a command that every couple of seconds, queries ES to see if it's ready, and if it is, continue on with the `python alert_worker.py -c alert_worker.conf` command (similar to https://github.com/scoringengine/scoringengine/blob/master/docker/wait-for-port.sh), but that's an improvement that can be made down the line

Reply:
Welp, disregard previous post, because I had typed it up before you posted about 'wait-for-it', so similar idea.

Reply:
i added a curl command for now